src/components/FilterSearch.tsx

```diff
@@ -130,7 +132,8 @@ export function FilterSearch({
     handleDropdownEvent(value, itemData, true);
   }, [handleDropdownEvent]);
 
-  const handleToggleDropdown = useCallback((isActive, _prevValue, value, _index, itemData) => {
+  const handleToggleDropdown =
```

Comment:
when clicking out of the dropdown without selecting a filter, should we reset the focused index? not sure if it's the desired behavior to have the dropdown remember where you were last when arrowing through